frontend/src/components/InputBox/InputBox.js

```diff
@@ -16,6 +16,7 @@ export default function InputBox({ feedback, setFeedback }) {
       <input
         type="text"
         className="feedback-input"
+        placeholder="Your feedback goes here"
```

Comment:
Above the InputBox says **"comment"** here says **"feedback"**_(even the description says **"messages"**)_. I think there should be more consistency with the language we use.

I recommend the title of the InputBox should be: **"Teacher Comment"**
And placeholder should be: **"Your comment goes here..."** 

Example:
![Screenshot 2022-02-11 005656](https://user-images.githubusercontent.com/35348015/153562956-e64c75c5-a00a-4635-ac36-1759edcf9768.png)
_NOTE: the `h1` css needed to be changed to 20% for this screenshot example._

This is not a required change, just my opinion. What are your thoughts on this?

Reply:
Totally agree with you. I think you meant to say h2, which I have changed to 20%. I have adjusted the changes. Thank you for checking it out.